packages/platform/libraries/index.ts

```diff
@@ -2,8 +2,11 @@ import * as newBookingMethods from "@calcom/features/bookings/lib/handleNewBooki
 import { getPublicEvent } from "@calcom/features/eventtypes/lib/getPublicEvent";
 import * as instantMeetingMethods from "@calcom/features/instant-meeting/handleInstantMeeting";
 import { updateHandler as updateScheduleHandler } from "@calcom/trpc/server/routers/viewer/availability/schedule/update.handler";
+import getAllUserBookings from "@calcom/trpc/server/routers/viewer/bookings/getAllUserBookings";
 import { getAvailableSlots } from "@calcom/trpc/server/routers/viewer/slots/util";
 
+import getBookingInfo from "../../../apps/web/modules/bookings/lib/getBookingInfo";
```

Comment:
cal we use absolute "@calcom/apps/..." import instead of relative "../../../" ?

Reply:
Yeah I tried this at first but for some reason it just doesn't seem to work :(